lib/ProductOpener/Tags.pm

```diff
@@ -195,6 +195,8 @@ use MIME::Base64 qw(encode_base64);
 use POSIX qw(strftime);
 use LWP::UserAgent ();
 use Encode;
+use IO::Compress::Gzip qw(gzip $GzipError);
+use IO::Uncompress::AnyInflate qw(anyinflate $AnyInflateError) ;
```

Comment:
Any (no pun intended) reason to prefer `AnyInflate` over `IO::Uncompress::Gunzip` (the symmetric library to `IO::Compress::Gzip`, if I understand correctly?) here?